addons/medical/CfgEden.hpp

```diff
@@ -9,8 +9,8 @@ class Cfg3DEN {
             };
         };
         class GVAR(isMedicControl): Title {
-            attributeLoad = "(_this controlsGroupCtrl 100) lbsetcursel (((_value + 1) min 3) max 0);";
-            attributeSave = "(missionnamespace getvariable ['ace_isMedic_temp',0]) - 1;";
+            attributeLoad = "(_this controlsGroupCtrl 100) lbSetCurSel (((_value + 1) min 3) max 0);";
+            attributeSave = "(_this getVariable ['ace_isMedic_temp', 0]) - 1;";
```

Comment:
@PabstMirror to your point about the rank module, I'm really not sure. The rank (and stance) toolboxes are stored as bonafide unit attributes, so I'm going to speculate and say BI is cheating and treats those slightly differently.